radio/src/opentx.cpp

```diff
@@ -1122,8 +1126,13 @@ void getADC()
     // Variables mapping:
     //   * <in> = v
     //   * <out> = s_anaFilt[x]
+#if defined(FLYSKY_HALL_STICKS)
+    uint32_t previous = s_anaFilt[x] / JITTER_ALPHA;
```

Comment:
Same here

Reply:
Thx. Learned again something. But as I changed now to 32-bit always, it can stay as uint32_t.